cmd/govim/main.go

```diff
@@ -197,6 +197,14 @@ type govimplugin struct {
 	applyEditsCh   chan applyEditCall
 	applyEditsLock sync.Mutex
 
+	// popupMenus is a map of active popups where the user can make a selection. It is keyed
```

Comment:
Can you clarify, how can there be more than one active popup?

Reply:
Active is probably not the best word. What we need to handle is multiple ShowMessageRequest calls from gopls. And it could, in theory, happen during a "suggested fixes" selection. I still have to figure out the best way to queue them (or if we just should close any open menu popup if we receive a ShowMessageRequest call).